dropwizard-jobs-core/src/main/java/de/spinscale/dropwizard/jobs/JobManager.java

```diff
@@ -218,34 +225,58 @@ protected Trigger executeNowTrigger() {
         return TriggerBuilder.newTrigger().startNow().build();
     }
 
+    private void createScheduler() throws SchedulerException {
+        // backwards compatibility - only use this when something is provided in
+        // the config
+        if (configuration.getQuartzConfiguration().isEmpty()) {
+            scheduler = StdSchedulerFactory.getDefaultScheduler();
+            return;
+        }
+        
+        StdSchedulerFactory factory = new StdSchedulerFactory();
+        factory.initialize(createProperties());
+        scheduler = factory.getScheduler();
+    }
+
+    private Properties createProperties() {
+        Properties properties = new Properties();
+        properties.putAll(configuration.getQuartzConfiguration());
+        return properties;
+    }
+
     private void logAllOnApplicationStopJobs() {
         log.info("Jobs to run on application stop:");
 
-        Arrays.stream(this.jobs)
-                .filter(job -> job.getClass().isAnnotationPresent(OnApplicationStop.class))
-                .map(job -> job.getClass())
-                .forEach(clazz -> log.info("   " + clazz.getCanonicalName()));
+        Arrays.stream(this.jobs).filter(job -> job.getClass().isAnnotationPresent(OnApplicationStop.class))
+                .map(job -> job.getClass()).forEach(clazz -> log.info("   " + clazz.getCanonicalName()));
     }
 
     private void createOrUpdateJob(JobKey jobKey, Class<? extends org.quartz.Job> clazz, Trigger trigger,
-                                   boolean requestsRecovery, boolean storeDurably) throws SchedulerException {
+            boolean requestsRecovery, boolean storeDurably) throws SchedulerException {
         JobBuilder jobBuilder = JobBuilder.newJob(clazz).withIdentity(jobKey).requestRecovery(requestsRecovery)
                 .storeDurably(storeDurably);
 
         if (!scheduler.checkExists(jobKey)) {
-            // if the job doesn't already exist, we can create it, along with its trigger. this prevents us
-            // from creating multiple instances of the same job when running in a clustered environment
+            // if the job doesn't already exist, we can create it, along with
+            // its trigger. this prevents us
+            // from creating multiple instances of the same job when running
+            // in
+            // a clustered environment
```

Comment:
yea - could you guys provide me your `.settings` folder in eclipse? then I'd use your code formatting settings and clean up this mess